pkg/api/handlers/libpod/play.go

```diff
@@ -66,9 +68,15 @@ func PlayKube(w http.ResponseWriter, r *http.Request) {
 		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "unable to create tempfile"))
 		return
 	}
-	defer os.Remove(tmpfile.Name())
+	defer func() {
+		if err := os.Remove(tmpfile.Name()); err != nil {
+			logrus.Error(err)
```

Comment:
```suggestion
			logrus.Warn(err)
```
Operation is going to fail, but not the service.